### PR TITLE
UKTS-7 - Special Endorsements - Basic Structure

### PR DESCRIPTION
**Pull request by AxonC**
- Stores a special endorsement
- Add the ability to grant a given user_id a special endorsement
- Associates positions with endorsements (endpoints to come in a later story)

### CODE REVIEW

File: graphql/endorsement.graphql [hunk 1]
```diff
@@ -0,0 +1,39 @@
+extend type Query {
+    specialEndorsements: [SpecialEndorsement!]! @all
+    specialEndorsement(id: ID @eq): SpecialEndorsement @find
+    specialEndorsementsByUser(user_id: ID @eq): Assignment @find
```
Comment (atoff) [inline graphql/endorsement.graphql]: this should return a list of SpecialEndorsement, not a single assignment pivot

**Review by atoff — CHANGES_REQUESTED** [review]

**Comment by AxonC** [comment]
Putting this on hold for now.

File: graphql/endorsement.graphql (hunk 1 above)
Comment (AxonC) [reply graphql/endorsement.graphql]: Now been addressed :)

**Comment by atoff** [comment]
@AxonC Don't mind approving this as is, although still iffy about the test override I did.

**Comment by AxonC** [comment]
> 
> 
> @AxonC Don't mind approving this as is, although still iffy about the test override I did.

I have a plan!

**Review by atoff — APPROVED** [review]

**Review by CalumTowers — APPROVED** [review]
We’ll need to think about how we link endorsements with sessions and approvals in the future but this makes for a perfect start in what is (actually) a fairly complex portion of the system 👍

**Pull request merged**